python/cudf_polars/cudf_polars/streaming/expressions.py

```
@@ -180,7 +180,7 @@ def _decompose_unique(
         A mapping from unique nodes in the new graph to associated
         partitioning information.
     """
-    from cudf_polars.experimental.distinct import lower_distinct
+    from cudf_polars.streaming.distinct import lower_distinct
```

Ruff (0.15.13)

[warning] `import` should be at the top-level of a file

(PLC0415)